PowerShell/ScubaGear/baselines/exo.md

```diff
@@ -113,20 +113,13 @@ points.
 ### Policies
 
 #### MS.EXO.2.1v1
-A list of approved IP addresses for sending mail SHALL be maintained.
+An SPF policy SHALL be published for each domain that fails all non-approved senders.
```

Comment:
description of the change notes that MS.EXO.2.1v1 will be deleted and changes will be applied to MS.EXO.2.2v1. 

If this is still the case I would remove all references to MS.EXO.2.1v1 and bump MS.EXO.2.2v1 to MS.EXO.2.2v2 with the updated policy, last modified date, and instructions.

Reply:
the subsequent changes to reference to the new MS.EXO.2.2v2 will also need to be reflected in the exoconfig.rego, exoconfig_02_test.rego, and exo.testplan.yaml files

Reply:
Relatedly, "An SPF policy SHALL be published..." appears now as MS.EXO.2.1.v1 in the markdown. Shouldn't it still be MS.EXO.2.2, but v2?

Reply:
Modified MS.EXO.2.2v2 (deleted MS>EXO.2.1v1)